robocop/config.py

```diff
@@ -77,7 +77,7 @@ def validate_regex(pattern: str) -> Pattern:
     try:
         return re.compile(pattern) if pattern is not None else None
     except re.error:
-        print("ups!")  # TODO
```

Comment:
It wasn't clear error message? :D And it shouldn't print - maybe you could reraise it as ConfigGeneralError or ValueError so it will stop the execution

Reply:
Updated with `ConfigGeneralError`